README.md

```diff
@@ -479,84 +473,87 @@ Which means that our contract was deployed and now we can invoke it.
-                  "state" : {
-                     "type" : "Array",
-                     "value" : [
-                        {
-                           "type" : "ByteArray",
-                           "value" : "56616c756520726561642066726f6d2073746f72616765"
-                        }
-                     ]
+            "contract" : "0x5776525e8c2c4ebccca99e71ee852e3d1bd073f9",
+            "state" : {
+               "type" : "Array",
+               "value" : [
+                  {
+                     "type" : "ByteString",
+                     "value" : "VmFsdWUgcmVhZCBmcm9tIHN0b3JhZ2U="
                   }
-               },
-               {
-                  "contract" : "0x85cf2075f3e297d489ff3c4c1745ca80d44e2a68",
-                  "state" : {
-                     "type" : "Array",
-                     "value" : [
-                        {
-                           "type" : "ByteArray",
-                           "value" : "53746f72616765206b6579206e6f7420796574207365742e2053657474696e6720746f2031"
-                        }
-                     ]
+               ]
+            }
+         },
+         {
+            "contract" : "0x5776525e8c2c4ebccca99e71ee852e3d1bd073f9",
+            "state" : {
+               "type" : "Array",
+               "value" : [
+                  {
+                     "value" : "U3RvcmFnZSBrZXkgbm90IHlldCBzZXQuIFNldHRpbmcgdG8gMQ==",
+                     "type" : "ByteString"
                   }
-               },
-               {
-                  "state" : {
-                     "value" : [
-                        {
-                           "value" : "4e65772076616c7565207772697474656e20696e746f2073746f72616765",
-                           "type" : "ByteArray"
-                        }
-                     ],
-                     "type" : "Array"
-                  },
-                  "contract" : "0x85cf2075f3e297d489ff3c4c1745ca80d44e2a68"
-               }
-            ],
-            "stack" : [
-               {
-                  "value" : "1",
-                  "type" : "Integer"
-               }
-            ]
+               ]
+            }
+         },
+         {
+            "state" : {
+               "value" : [
+                  {
+                     "type" : "ByteString",
+                     "value" : "TmV3IHZhbHVlIHdyaXR0ZW4gaW50byBzdG9yYWdl"
+                  }
+               ],
+               "type" : "Array"
+            },
+            "contract" : "0x5776525e8c2c4ebccca99e71ee852e3d1bd073f9"
          }
-      ],
-      "txid" : "0x6f27f523da9c71297f4a81a254274b0c2a78f893b81c500429be6230254be0bf"
-   },
-   "jsonrpc" : "2.0"
+      ]
+   }
 }
 ```
 Pay attention to `notification` field. It contains messages, which where passed to `runtime.Notify` method.
-This one contains hexadecimal byte arrays which can be decoded into 3 messages:
+This one contains base64 byte arrays which can be decoded into 3 messages:
```

Comment:
Maybe we should show some `echo string | base64 -d` here.

Reply:
Did you mean adding a couple of words on how to use `echo string | base64 -d` or just putting application log with human-readable fields?

Reply:
We should show how to decipher these base64 values, application log is application log, it shouldn't be changed in any way.